### ISSUE TITLE
Inputmask decimal alias with digits option on 0.00 initialized fields.

### ISSUE THREAD

**Issue by sas1973** [RobinHerbots/Inputmask#243]
I have tried different ways to do this but i haven't had a success, i am not sure if this is a issues or i am just missing something.

Problem/Requirement: 
I have decimal input fields that can accept negative values, and the decimal digits vary from form to form(i mean some fields has 3 decimals some has 2). The server sends the fields initialized to 0.00 (e.g. for a 2 decimal digits field) or its current value if it exists. 
If user decides to enter new data or change the field value, say for example 15 in the field without entering any decimal or digits it should allow them overwrite on the 0 without user having to backspace and clear the 0's or the existing value of the field.  
Also the caret should be positioned to left (i.e. before the decimal part).If the user enter only the whole number like 15 (for e.g. in a decimal field that expects 2 decimal places) then on form submit the field value should be posted as 15.00. if user enters 15.1 then it should be posted as 15.10. 

To summarize requirements:
Decimal Value
Allow Negatives
Allow Overwriting on existing data without user having to backspace to.
Always fill the decimal digits to 0 if incomplete so field value is posted with decimal digits. 
Caret should always be positioned left or before decimal. 

I have so far used decimal alias in my code to allow -ve with digits, this satisfy my 1,2 requirements. 

$("#FIELD_NUM_10").inputmask('decimal', { digits: 2});

![image](https://f.cloud.github.com/assets/3344621/572793/d39265b8-c7a2-11e2-918c-457277dbdfe0.png)

Need help on last two requirements 

Thanks

**Comment by RobinHerbots** [comment]
@sas1973, 

I will have a look at the initial positioning.

**Comment by RobinHerbots** [comment]
@sas1973 , 

The positioning should be fixed in the latest version, but there is still an issue in checkval when setting an initial value.

I will fix that soon.

**Comment by RobinHerbots** [comment]
@sas1973 , 

For the overwrite set  insertMode: false, option
